blocks/collapse/collapse.css

```diff
@@ -15,26 +56,80 @@
   height: var(--spacing-vertical-xxl);
   opacity: 1;
   transition: opacity .2s ease-in-out;
-  background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0), var(--bg-color-primary) 60%)
+  background-image: linear-gradient(to bottom, rgb(255 255 255 / 0%), var(--bg-color-primary) 60%);
+}
+
+a {
+  font-family: var(--font-family-frutiger);
+  font-weight: 500;
+  font-size: 1rem;
+  color: var(--button-tertiary-text-default);
+  position: relative;
+  text-decoration: none;
 }
 
-.text-block--expanded .text-block__expandable-area::after {
+ p.button-container {
+   margin-top: var(--spacing-vertical-s);
+ }
```

Comment:
if these are global styles and not limited to `collapse` block, we should move them to a common location like `styles.css`. And if specific to collapse only, we should add `collapse` class in the selector to prevent them from leaking across other places.